### ISSUE TITLE
Not working for http requests for Angular 8.1.0

### ISSUE THREAD

**Issue by quraishishoaib** [mpalourdio/ng-http-loader#127]
Loader not working for http requests for Angular version 8.1.0 and it also not showing any error. When I manually show spinner it works but not for http requests. Following is my package.json

```
{
  "name": "client-app",
  "version": "0.0.0",
  "scripts": {
    "ng": "ng",
    "start": "ng serve",
    "build": "ng build",
    "prod-build": "ng build --prod",
    "test": "ng test",
    "lint": "ng lint",
    "e2e": "ng e2e"
  },
  "private": true,
  "husky": {
    "hooks": {
      "pre-commit": "npm run prod-build && npm run lint"
    }
  },
  "dependencies": {
    "@angular/animations": "~8.1.0",
    "@angular/cdk": "^8.1.0",
    "@angular/common": "~8.1.0",
    "@angular/compiler": "~8.1.0",
    "@angular/core": "~8.1.0",
    "@angular/forms": "~8.1.0",
    "@angular/material": "^8.1.0",
    "@angular/platform-browser": "~8.1.0",
    "@angular/platform-browser-dynamic": "~8.1.0",
    "@angular/router": "~8.1.0",
    "@ng-select/ng-select": "^3.0.6",
    "@ngx-translate/core": "^11.0.1",
    "@ngx-translate/http-loader": "^4.0.0",
    "angular-datatables": "^8.0.0",
    "angular-font-awesome": "^3.1.2",
    "bootstrap": "^4.3.1",
    "datatables.net": "^1.10.19",
    "datatables.net-bs4": "^1.10.19",
    "datatables.net-dt": "^1.10.19",
    "file-saver": "^2.0.2",
    "font-awesome": "^4.7.0",
    "hammerjs": "^2.0.8",
    "husky": "^3.0.2",
    "jquery": "^3.4.1",
    "moment": "^2.24.0",
    "ng-http-loader": "^6.0.1",
    "ngx-bootstrap": "^5.1.1",
    "ngx-daterangepicker-material": "^2.1.8",
    "ngx-material-timepicker": "^4.0.2",
    "ngx-toastr": "^10.0.4",
    "rxjs": "^6.5.0",
    "tslib": "^1.9.0",
    "zone.js": "~0.9.1"
  },
  "devDependencies": {
    "@angular-devkit/build-angular": "^0.801.3",
    "@angular/cli": "~8.1.0",
    "@angular/compiler-cli": "~8.1.0",
    "@angular/language-service": "~8.1.0",
    "@types/datatables.net": "^1.10.17",
    "@types/jasmine": "~3.3.8",
    "@types/jasminewd2": "~2.0.3",
    "@types/jquery": "^3.3.31",
    "@types/node": "~8.9.4",
    "codelyzer": "^5.0.0",
    "jasmine-core": "~3.4.0",
    "jasmine-spec-reporter": "~4.2.1",
    "karma": "~4.1.0",
    "karma-chrome-launcher": "~2.2.0",
    "karma-coverage-istanbul-reporter": "~2.0.1",
    "karma-jasmine": "~2.0.1",
    "karma-jasmine-html-reporter": "^1.4.0",
    "protractor": "~5.4.0",
    "ts-node": "~7.0.0",
    "tslint": "~5.15.0",
    "typescript": "~3.4.3"
  }
}
```

**Comment by mpalourdio** [comment]
Definitely an integration problem IMO, but without any code, i just cannot help you or fix a potential bug.(if any)

Feel free to re-open once you have a minimum reproducible case I can easily run.

**Comment by quraishishoaib** [comment]
Thanks @mpalourdio I really appreciate your work. I just solved the issue it was related to #51
